avocado/utils/multipath.py

```diff
@@ -115,6 +115,48 @@ def get_paths(wwid):
     return paths
 
 
+def get_multipath_details():
+    """
+    Get multipath details as a dictionary, as given by the command:
+    multipathd show maps json
+
+    :return: Dictionary of multipath output in json format.
+    """
+    mpath_op = process.system_output("multipathd show maps json", shell=True)
```

Comment:
Why is shell needed here? Also, I remember that only recent versions of multipathd support json.  It's less convenient, but I'd try parsing the traditional output instead.